### ISSUE TITLE
Component: p-inputNumber - [Accessibility - Screen reader - Voiceover on iOS]

### ISSUE THREAD

**Issue by jljohnson001** [primefaces/primeng#17389]
### Describe the bug

Using iOS Voiceover in Safari, user cannot complete PrimeNg InputNumber. See video:

https://github.com/user-attachments/assets/4ef1ea69-5b10-4f9f-aaa8-ba7f035e9f79

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/inputnumber

### Environment

PrimeNg 17.x and also 18, 19.

### Angular version

^18.0.4

### PrimeNG version

v19

### Browser(s)

Safari latest

### Steps to reproduce the behavior

1. Go to https://v17.primeng.org/inputnumber or latest version
2. On an iOS device, turn Voiceover on
3. Using Voiceover navigate to any of the InputNumber fields on the page
4. Following Voiceover instructions, swipe up or down to change the number
5. Observe: numbers available are only Infinity or -Infinity OR the min/max set for the input (no numbers in between) 

### Expected behavior

User can enter in input when using Voiceover on iOS on a PrimeNg InputNumber field. Or toggle buttons to increase or decrease number input.

**Comment by mertsincan** [comment]
Could you please try it with v19.x?
